application/src/main/resources/application.yml

```diff
@@ -42,10 +42,19 @@ springdoc:
 #server:
 #  ssl:
 #    enabled: false
+#    client-auth: <client authentication>
+# SSL configuration using certificate stores
 #    key-store: <path to keystore>
 #    key-store-password: <password>
 #    key-store-type: <keystore type>
 #    key-alias: <key alias>
 #    key-password: <key password>
-# Rate at which certificate are reloaded automatically
+# SSL configuration using certificates in PEM format
+#    certificate: <path to certificate>
+#    certificate-private-key: <path to certificate private key>
+#    key-store-password: <key password>
```

Comment:
what is key-store-password used for and what's the difference with key-password?

Reply:
As commented above, the documentation describes it in different fields.

Reply:
Ah, key-store-password is the password for the keystore itself while key-password is the password for the private key in the keystore. So both are connected to keystore. Please move key-store-password to where the other keystore related configuration is.

Reply:
btw, this "key-store-password" is already defined on line 48.

Reply:
Yes it is defined, but I thought it was important to mention because without it it wasn't working. But with version 2.7.2, with it empty works :)